packages/react-native/React/CoreModules/RCTRedBox.mm

```diff
@@ -592,11 +606,8 @@ - (void)showErrorMessage:(NSString *)message
     [[self->_moduleRegistry moduleForName:"EventDispatcher"] sendDeviceEventWithName:@"collectRedBoxExtraData"
                                                                                 body:nil];
 #pragma clang diagnostic pop
-
     if (!self->_window) {
-      self->_window = [[RCTRedBoxWindow alloc] initWithFrame:[UIScreen mainScreen].bounds
-                                          customButtonTitles:self->_customButtonTitles
-                                        customButtonHandlers:self->_customButtonHandlers];
+        self->_window = [[RCTRedBoxController alloc] initWithCustomButtonTitles:self->_customButtonTitles customButtonHandlers:self->_customButtonHandlers];
```

Comment:
Can you rename `_window` to `_controller` too?

Reply:
Renamed!